vcpkg.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": [
+    "openssl"
```

Comment:
Maybe later, should there be a less invasive dependency here?
I mean this is bound to NOT work on the Web platform, so maybe something smaller/self contained?

Reply:
good point. Lets investigate this when we add wasm to the extension-workflows pipeline